recipe/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "momepy" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/momepy-{{ version }}.tar.gz
+  sha256: 413ed9221dc4b2190621f8cb37789f09ea882f7e81dfbffd5731a22da6e61bab
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [py<310 or s390x]
+
+requirements:
+  host:
+    - python
+    - setuptools >=61.0
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python
+    - geopandas >=0.12.0
+    - libpysal >=4.6.0
```

```suggestion
    - libpysal {{ libpysal }}
```
This might work, but I'm going to try it first before I tell you wrong.

---

I think adding the above mentioned to `run` and adding the following to the `conda_build_config.yaml` **_may_** possibly do the trick, but I would love more eyes on this solution.

```
libpysal:
  - 4.10
```